source/agora/common/crypto/ECC.d

```diff
@@ -103,6 +135,21 @@ public struct Scalar
         return this.data.toString();
```

Comment:
This is problematic now

Reply:
Yes, and these `string toString ()` methods are used in other `unittest`s.
Would it be a good idea to remove the duplicate and use `scope void delegate` everywhere?